cmd/influxd/inspect/dump_wal/dump_wal.go

```diff
@@ -43,20 +43,20 @@ It has two modes of operation, depending on the --find-duplicates flag.
 	return cmd
 }
 
-func (dumpWAL *dumpWALCommand) run(args []string) error {
+func (dumpWAL *dumpWALCommand) run(cmd cobra.Command, args []string) error {
 
 	// Process each WAL file.
 	for _, path := range args {
-		if err := dumpWAL.processWALFile(path); err != nil {
+		if err := dumpWAL.processWALFile(cmd, path); err != nil {
 			return err
 		}
 	}
 	return nil
 }
 
-func (dumpWAL *dumpWALCommand) processWALFile(path string) error {
+func (dumpWAL *dumpWALCommand) processWALFile(cmd cobra.Command, path string) error {
 	if filepath.Ext(path) != "."+tsm1.WALFileExtension {
-		fmt.Fprintf(os.Stderr, "invalid wal file path, skipping %s\n", path)
+		cmd.Printf("invalid wal file path, skipping %s\n", path)
```

Comment:
You can do a `return fmt.Errorf()` here instead, and Cobra will handle it, but it will return an error instead of `nil` if that's okay

Reply:
So for this one I think I'll keep it as is, just because returning nil rather than an error allows one (or more) invalid file paths to be skipped, while still being able to continue processing the valid ones.